packages/ifpack2/src/Ifpack2_Details_Fildl_decl.hpp

```diff
@@ -64,9 +64,9 @@ class Fildl : public FastILU_Base<Scalar, LocalOrdinal, GlobalOrdinal, Node>
   public:
     typedef FastILU_Base<Scalar, LocalOrdinal, GlobalOrdinal, Node> Base;
     typedef typename Base::TRowMatrix TRowMatrix;
-    typedef typename Base::ScalarArray ScalarArray;
-    typedef typename Base::ConstScalarArray ConstScalarArray;
-    typedef FastILDLPrec<LocalOrdinal, Scalar, typename Base::execution_space> LocalFILDL;
+    typedef typename Base::ImplScalar ImplScalar;
+    typedef typename Base::ImplScalarArray ImplScalarArray;
```

Comment:
Is there still constness assumed with this array? If so, it may be helpful to keep that in the typedef for readability